enterprisemanagementv1/enterprise_management_v1_integration_test.go

```diff
@@ -19,780 +19,405 @@
+			if err != nil {
+				Skip("Error loading service properties, skipping tests: " + err.Error())
+			}
+			serviceURL = testConfig["URL"]
+			if serviceURL == "" {
+				Skip("Unable to load service URL configuration property, skipping tests")
+			}
 
-		err = json.Unmarshal(body, &results)
-		Expect(err).To(BeNil())
+			enterpriseId = testConfig["ENTERPRISE_ID"]
+			Expect(enterpriseId).ToNot(BeEmpty())
 
-		res := results["resources"].([]interface{})
+			accountId = testConfig["ACCOUNT_ID"]
+			Expect(accountId).ToNot(BeEmpty())
 
-		z := res[0].(map[string]interface{})
+			accountIamId = testConfig["ACCOUNT_IAM_ID"]
+			Expect(accountIamId).NotTo(BeEmpty())
 
-		var ok bool
-		activationId, ok = z["id"].(string)
-		Expect(ok).To(BeTrue())
+			fmt.Fprintf(GinkgoWriter, "Service URL: %s\n", serviceURL)
+			shouldSkipTest = func() {}
+		})
 	})
 
-	It("Successfully activate account", func() {
-		shouldSkipTest()
-
-		apiUrl := testConfig["AM_HOST"]
-		resource := "/coe/v2/accounts/verify"
-
-		u, err := url.ParseRequestURI(apiUrl)
-		Expect(err).To(BeNil())
-
-		u.Path = resource
-		urlStr := u.String()
+	Describe(`Client initialization`, func() {
+		BeforeEach(func() {
+			shouldSkipTest()
+		})
+		It("Successfully construct the service client instance", func() {
 
-		url := urlStr
+			enterpriseManagementServiceOptions := &enterprisemanagementv1.EnterpriseManagementV1Options{}
 
-		req, err := http.NewRequest("GET", url, nil)
-		Expect(err).To(BeNil())
+			enterpriseManagementService, err = enterprisemanagementv1.NewEnterpriseManagementV1UsingExternalConfig(enterpriseManagementServiceOptions)
 
-		err = amAuth.Authenticate(req)
-		Expect(err).To(BeNil())
+			core.SetLogger(core.NewLogger(core.LevelDebug, log.New(GinkgoWriter, "", log.LstdFlags), log.New(GinkgoWriter, "", log.LstdFlags)))
+			enterpriseManagementService.EnableRetries(4, 30*time.Second)
 
-		q := req.URL.Query()
-		q.Add("token", activationId)
-		q.Add("email", email)
-		req.URL.RawQuery = q.Encode()
-
-		// Send req using http Client
-		client := &http.Client{}
-
-		resp, err := client.Do(req)
-		Expect(err).To(BeNil())
-
-		body, err := ioutil.ReadAll(resp.Body)
-		Expect(err).To(BeNil())
-		Expect(body).ToNot(BeNil())
+			Expect(err).To(BeNil())
+			Expect(enterpriseManagementService).ToNot(BeNil())
+			Expect(enterpriseManagementService.Service.Options.URL).To(Equal(serviceURL))
+		})
 	})
 
-	It("Successfully get account", func() {
-		shouldSkipTest()
-
-		apiUrl := testConfig["AM_HOST"]
-		resource := "/coe/v2/accounts/" + account_id
-		u, err := url.ParseRequestURI(apiUrl)
-		Expect(err).To(BeNil())
-
-		u.Path = resource
-		urlStr := u.String()
-
-		url := urlStr
-
-		req, err := http.NewRequest("GET", url, nil)
-		Expect(err).To(BeNil())
+	Describe(`CreateAccountGroup - Create an account group`, func() {
+		BeforeEach(func() {
+			shouldSkipTest()
+		})
+		It(`CreateAccountGroup(createAccountGroupOptions *CreateAccountGroupOptions)`, func() {
```

Comment:
I think a cleaner way to create 2 account groups would be to have two separate It() blocks, with each one creating a single account group.